### ISSUE TITLE
Difference between "Heading Numbers" and "Markdown Heading Numbers"?

### ISSUE THREAD

**Issue by chronoguy** [jordan2175/markdown-tools#12]
Thank you for publishing this Add-On. It's hard to unterstand why Google Docs are missing such a basic functionality out of the box.

Could you please elaborate on the differences between "_Heading Numbers_" and "_Markdown Heading Numbers_" functions of this plug-in? It may be trivial for folks who have looked into Google Docs internals, but as a "simple" user I do not grasp the difference, nor which of the two I "want". 

Thank you!

**Comment by chronoguy** [comment]
Looks like the "Markdown Heading Numbers" just inserts a corresponding number of `#` in front of each heading. I am not sure of the scenario for it, but I am certain it is useful in some cases. Is this the only difference or are there additional differences?

Thanks!

**Comment by jordan2175** [comment]
Hi, thanks for reaching out. I originally wrote this plugin to enable myself and a few others to work together via Google docs on documents written in Markdown format. Thus you have the ability to add one or more # hash marks to headings, depending on the heading level. You also have the ability to create a table of contents in markdown format. After I had this working, i got a request from a friend to add support for normal heading numbers. Since the code was nearly identical, I just added that support.  But clearly the plugin has become popular. I have thought about creating a version 2 of the plugin with a lot more advanced features, I just have been crazy busy on other projects and have not finished it yet.  

So in short, the markdown headings are for documents written in markdown format, think Github README.md files.
